### DISCUSSION TITLE
Llama2 本地模型的配置方法

### DISCUSSION THREAD

**Discussion by woyin** [nextai-translator/nextai-translator#1341] [Q&A]
已经将Llama2跑了起来，如下图：
<img width="929" alt="image" src="https://github.com/openai-translator/openai-translator/assets/35319250/98dac332-0874-4daa-939e-8839a17cb688">
OpenAI Translator配置如下：
<img width="606" alt="image" src="https://github.com/openai-translator/openai-translator/assets/35319250/d6d3a97a-2ad3-4e8c-a87a-6caf53aaa73c">
翻译后结果如下：
<img width="614" alt="image" src="https://github.com/openai-translator/openai-translator/assets/35319250/db921846-eaaf-498f-9268-fdbe4938a55f">
显示一直在翻译，但是出现了那个👍，但就是没有任何东西出来。

**Comment by yetone** [comment]
你在 terminal 里试一下 `ollama run llama2`

**Reply by woyin** [reply]
我已经跑过了，而且在terminal中可以正常的交互。具体输出如下：
<img width="941" alt="image" src="https://github.com/openai-translator/openai-translator/assets/35319250/ce75fc34-e3ac-4ec2-b3e8-0429503a4333">

**Reply by woyin** [reply]
即使这边跑起来，在app哪里的反应依然是和之前一样。

**Reply by yetone** [reply]
@woyin 把翻墙软件或者防火墙彻底关闭一下试一试呢

**Reply by woyin** [reply]
我天，果然是因为Clash的问题，用TUN方式翻墙，难道这会导致路径变化？
针对我这种需要全天全局代理的场景，是不是只能关闭代理后使用了？

**Reply by yetone** [reply]
我没用过不太清楚 clash 如何配置，你可以搜一下如何忽略掉 localhost/127.0.0.1 的代理

**Reply by yetone** [reply]
我刚刚在本地试了一下 clashx pro，开了全局代理，没复现这个问题，可能是你的 clash 有某个配置导致的

**Reply by woyin** [reply]
感谢您，谢谢。

**Comment by lukaschan** [comment]
加条规则试试：- IP-CIDR,127.0.0.1/16,DIRECT,no-resolve

**Reply by woyin** [reply]
感谢。
